src/App.tsx

```diff
@@ -15,35 +16,117 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  alpha = 'alpha',
+  length = 'length',
+  empty = ''
+}
+
 export const App: React.FC = () => {
+  const [sortField, setSortField] = useState<SortType>(SortType.empty)
```

Comment:
```suggestion
  const [sortField, setSortField] = useState(SortType.empty)
```
ts already knows that `sortField` will be `SortType` since you set initial value of that type